pkgs/development/python-modules/tableaudocumentapi/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "tableaudocumentapi";
+  version = "0.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fc6d44b62cf6ea29916c073686e2f9f35c9902eccd57b8493f8d44a59a2f60d9";
+  };
+
+  # tests not inclued with release
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A Python module for working with Tableau files.";
```

```suggestion
    description = "A Python module for working with Tableau files";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

---

Fixed thanks for catching that.